### ISSUE TITLE
[Documentation] List Command

### ISSUE THREAD

**Issue by jonnyboyC** [KSP-KOS/KOS#2512]
It appears that the List command documentation is somewhat incorrect. The 4th bullet of [this list](https://github.com/KSP-KOS/KOS/blob/develop/doc/source/commands/list.rst#for-loop) indicates that you can do something like this:

    LIST ListKeyword FROM SomeVessel IN YourVariable.

A quick test in kOS with `List engines from ship in example`. Indicates a parser error. The [grammar files](https://github.com/KSP-KOS/KOS/blob/c2204c28741516f4845d463d56e59f34f0a22ece/src/kOS.Safe/Compilation/KS/kRISC.tpg#L200) does not appear to have this variant either. 

So it seems that this documentation should be updated to remove this variant of the list command

**Comment by Dunbaratu** [comment]
Yes this is an old feature that went away.  Apparently we didn't catch it everywhere in the docs.  Thanks for the report.
